rust/lance-table/src/rowids.rs

```diff
@@ -152,6 +152,27 @@ impl RowIdSequence {
         self.0.extend_from_slice(remaining_segments);
     }
 
+    /// Mask
```

Comment:
```suggestion
    /// Delete row ids by position
```
Or some other comment explaining the difference between this and `delete` (they look very similar)